### ISSUE TITLE
Predictions coming out NAN

### ISSUE THREAD

**Issue by sNIKersBar** [google-deepmind/graphcast#177]
I am trying to run the 0.25 degree model and make predictions for 48 hours ahead. 

I downloaded the weights and normalization 

<img width="247" height="131" alt="Image" src="https://github.com/user-attachments/assets/a43c7959-1b06-4488-a932-774fb2f791cc" /> 

This is my input 

<img width="567" height="499" alt="Image" src="https://github.com/user-attachments/assets/149c73c4-fea9-4c90-bdbf-1fe4d88c1de0" />

Targets

<img width="544" height="329" alt="Image" src="https://github.com/user-attachments/assets/263c15ce-acdd-4037-9470-4fb0528d261d" />

Forcings

<img width="558" height="305" alt="Image" src="https://github.com/user-attachments/assets/a065748f-a4b5-43ff-b762-7f3f4a0819b0" />

I am trying to make predictions like this:

`with open(r'/model/params/params_GraphCast - ERA5 1979-2017 - resolution 0.25 - pressure levels 37 - mesh 2to6 - precipitation input and output.npz', 'rb') as model:
    ckpt = checkpoint.load(model, graphcast.CheckPoint)

    params = ckpt.params
    state = {}
    model_config = ckpt.model_config
    task_config = ckpt.task_config

with open(r'/model/stats/stats_diffs_stddev_by_level.nc', 'rb') as f:
    diffs_stddev_by_level = xarray.load_dataset(f).compute()

with open(r'/model/stats/stats_mean_by_level.nc', 'rb') as f:
    mean_by_level = xarray.load_dataset(f).compute()

with open(r'/model/stats/stats_stddev_by_level.nc', 'rb') as f:
    stddev_by_level = xarray.load_dataset(f).compute()

def construct_wrapped_graphcast(model_config:graphcast.ModelConfig, task_config:graphcast.TaskConfig):
    predictor = graphcast.GraphCast(model_config, task_config)
    #predictor = casting.Bfloat16Cast(predictor)
    predictor = normalization.InputsAndResiduals(predictor, diffs_stddev_by_level = diffs_stddev_by_level, mean_by_level = mean_by_level, stddev_by_level = stddev_by_level)
    predictor = autoregressive.Predictor(predictor, gradient_checkpointing = True)
    return predictor

@hk.transform_with_state
def run_forward(model_config, task_config, inputs, targets_template, forcings):
    predictor = construct_wrapped_graphcast(model_config, task_config)
    return predictor(inputs, targets_template = targets_template, forcings = forcings)

def with_configs(fn):
    return functools.partial(fn, model_config = model_config, task_config = task_config)

def with_params(fn):
    return functools.partial(fn, params = params, state = state)

def drop_state(fn):
    return lambda **kw: fn(**kw)[0]

run_forward_jitted = drop_state(with_params(jax.jit(with_configs(run_forward.apply))))

class Predictor:

    @classmethod
    def predict(cls, inputs, targets, forcings) -> xarray.Dataset:
        predictions = rollout.chunked_prediction(run_forward_jitted, rng = jax.random.PRNGKey(0), inputs = inputs, targets_template = targets, forcings = forcings)
        return predictions`  

At first I get this error: dtype argument must be a NumPy dtype, but it is a <class 'numpy.dtype[bfloat16]'>.

So then I comment out (#predictor = casting.Bfloat16Cast(predictor)) 

The the model runs and I get the predictions but all the values are NAN

<img width="533" height="338" alt="Image" src="https://github.com/user-attachments/assets/77532889-f7c7-4896-9e47-888f01f20d70" />

I am using CPU only but I have lots of memory available and I was able to make a prediction for the same dates but on 1 degree data still using the 0.25 degree model's checkpoint. 

If anyone knows why the model is outputting all NAN I would really appreciate the help.

**Comment by sNIKersBar** [comment]
just noticed my forcings are wrong... I will change this and update this thread if it works.
